app/scripts/views/sign_up.js

```diff
@@ -36,6 +40,44 @@ define(function (require, exports, module) {
     return null;
   }
 
+  var actions = {
+    signIn: function (account, password) {
+      var self = this;
+      return this.user.signInAccount(account, password, this.relier, {
+        resume: this.getStringifiedResumeToken()
+      })
+      .then(function (account) {
+        actions.after.call(self);
+        return self.invokeBrokerMethod('afterSignIn', account);
+      })
+      .then(this.onSignInSuccess.bind(this, account));
+    },
+
+    signUp: function (account, password, preVerifyToken) {
+      var self = this;
+      return this.user.signUpAccount(account, password, this.relier, {
+        resume: this.getStringifiedResumeToken()
+      })
+      .then(function (account) {
+        actions.after.call(self);
+        if (preVerifyToken && account.get('verified')) {
```

Comment:
Think lines 63-63 should be moved to the [signup-success-mixin.js](https://github.com/mozilla/fxa-content-server/pull/3413/files#diff-3ec738e8179a9c67ea488ee0316b3ad2)? Would doing so break other views?